src/app/GitUI/Editor/FormFindInCommitFilesGitGrep.cs

```diff
@@ -1,10 +1,15 @@
 ﻿using GitCommands;
 using GitExtensions.Extensibility.Git;
+using GitUI.Editor;
+using ResourceManager;
 
 namespace GitUI;
 
 internal partial class FormFindInCommitFilesGitGrep : GitExtensionsDialog
 {
+    private readonly TranslationString _watermarkText = new("git-grep regular expression...");
```

Comment:
I always hesitate adding an explicit `TranslationString` instance.

Reply:
Perhaps move to https://github.com/gitextensions/gitextensions/blob/master/src/app/GitUI/TranslatedStrings.cs?